### ISSUE TITLE
[CI] EncryptedAzureBlobStoreRepositoryIntegTests classMethod failing

### ISSUE THREAD

**Issue by hendrikmuhs** [elastic/elasticsearch#73560]
started failing after #73493

**Build scan:**
https://gradle-enterprise.elastic.co/s/anui462zg63sy/tests/:x-pack:plugin:repository-encrypted:internalClusterTest/org.elasticsearch.repositories.encrypted.EncryptedAzureBlobStoreRepositoryIntegTests/classMethod

**Reproduction line:**
`null`

**Applicable branches:**
7.x

**Reproduces locally?:**
Didn't try

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.repositories.encrypted.EncryptedAzureBlobStoreRepositoryIntegTests&tests.test=classMethod

**Failure excerpt:**
```
java.security.AccessControlException: access denied ("java.io.FilePermission" "C:\Windows\system32\drivers\etc\hosts" "read")

  at java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
  at java.security.AccessController.checkPermission(AccessController.java:886)
  at java.lang.SecurityManager.checkPermission(SecurityManager.java:549)
  at java.lang.SecurityManager.checkRead(SecurityManager.java:888)
  at java.io.File.exists(File.java:823)
  at io.netty.resolver.HostsFileParser.locateHostsFile(HostsFileParser.java:59)
  at io.netty.resolver.HostsFileParser.parseSilently(HostsFileParser.java:85)
  at io.netty.resolver.DefaultHostsFileEntriesResolver.parseEntries(DefaultHostsFileEntriesResolver.java:75)
  at io.netty.resolver.DefaultHostsFileEntriesResolver.<init>(DefaultHostsFileEntriesResolver.java:37)
  at io.netty.resolver.HostsFileEntriesResolver.<clinit>(HostsFileEntriesResolver.java:28)
  at java.lang.Class.forName0(Class.java:-2)
  at java.lang.Class.forName(Class.java:264)
  at org.elasticsearch.repositories.azure.AzureRepositoryPlugin.lambda$static$0(AzureRepositoryPlugin.java:63)
  at java.security.AccessController.doPrivileged(AccessController.java:-2)
  at org.elasticsearch.repositories.azure.AzureRepositoryPlugin.<clinit>(AzureRepositoryPlugin.java:61)
  at java.lang.Class.forName0(Class.java:-2)
  at java.lang.Class.forName(Class.java:264)
  at org.elasticsearch.bootstrap.BootstrapForTesting.<clinit>(BootstrapForTesting.java:168)
  at org.elasticsearch.test.ESTestCase.<clinit>(ESTestCase.java:230)
  at java.lang.Class.forName0(Class.java:-2)
  at java.lang.Class.forName(Class.java:348)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$2.run(RandomizedRunner.java:623)

```

**Comment by fcofdez** [comment]
This is the same as #73539

**Comment by gwbrown** [comment]
Closing, as this is a duplicate of https://github.com/elastic/elasticsearch/issues/73539 (as noted above).
